src/main/scala/nl.knaw.dans.easy.validatebag/rules/ProfileVersion1.scala

```diff
@@ -35,19 +35,19 @@ object ProfileVersion1 {
 
     // bag-info.txt
     NumberedRule("1.2.1", containsFile(Paths.get("bag-info.txt"))),
-    NumberedRule("1.2.2", bagInfoContainsExactlyOneOf("BagIt-Profile-Version"), dependsOn = Some("1.2.1")),
-    NumberedRule("1.2.2", bagInfoElementIfExistsHasValue("BagIt-Profile-Version", versionNumber.toString), dependsOn = Some("1.2.2")),
-    NumberedRule("1.2.3", bagInfoContainsExactlyOneOf("BagIt-Profile-URI"), dependsOn = Some("1.2.1")),
-    NumberedRule("1.2.3", bagInfoElementIfExistsHasValue("BagIt-Profile-URI", versionUri), dependsOn = Some("1.2.3")),
-    NumberedRule("1.2.4", bagInfoContainsExactlyOneOf("Created"), dependsOn = Some("1.2.1")),
-    NumberedRule("1.2.4", bagInfoCreatedElementIsIso8601Date, dependsOn = Some("1.2.4")),
+    NumberedRule("1.2.2(a)", bagInfoContainsExactlyOneOf("BagIt-Profile-Version"), dependsOn = Some("1.2.1")),
+    NumberedRule("1.2.2(b)", bagInfoElementIfExistsHasValue("BagIt-Profile-Version", versionNumber.toString), dependsOn = Some("1.2.2(a)")),
+    NumberedRule("1.2.3(a)", bagInfoContainsExactlyOneOf("BagIt-Profile-URI"), dependsOn = Some("1.2.1")),
+    NumberedRule("1.2.3(b)", bagInfoElementIfExistsHasValue("BagIt-Profile-URI", versionUri), dependsOn = Some("1.2.3(a)")),
+    NumberedRule("1.2.4(a)", bagInfoContainsExactlyOneOf("Created"), dependsOn = Some("1.2.1")),
+    NumberedRule("1.2.4(b)", bagInfoCreatedElementIsIso8601Date, dependsOn = Some("1.2.4(a)")),
     NumberedRule("1.2.5", bagInfoContainsAtMostOneOf("Is-Version-Of"), dependsOn = Some("1.2.1")),
-    NumberedRule("1.2.6", bagInfoContainsExactlyOneOf("EASY-User-Account"), AIP, dependsOn = Some("1.2.1")),
-    NumberedRule("1.2.6", bagInfoDoesNotContain("EASY-User-Account"), SIP, dependsOn = Some("1.2.1")),
+    NumberedRule("1.2.6(a)", bagInfoContainsExactlyOneOf("EASY-User-Account"), AIP, dependsOn = Some("1.2.1")),
+    // 1.2.6(b) does not state restrictions, so it does not need checking
```

Comment:
Is this change correct? @janvanmansum

Reply:
Yes, it is, see: https://github.com/DANS-KNAW/easy-specs/blob/master/dans-bagit-profile/v1/dans-bagit-profile-v1.md#12-bag-infotxt under 6.